config/checkstyle_non_main_files_checks.xml

```diff
@@ -23,6 +23,12 @@
                value="^(?!(.*href=|.*http(s)?:|import |(.* )?package |.* files=|.*\.dtd)).{101,}$"/>
       <property name="message" value="Line should not be longer than 100 symbols"/>
     </module>
+    <module name="RegexpSingleline">
+        <property name="format" value="(mvn [^-])|(mvn \-(?!v)(?!-v)[^e])"/>
+        <property name="fileExtensions" value="sh, yml"/>
+        <property name="message" value="mvn execution should be with '-e' option
+                                        to produce error message in case of problem"/>
```

Comment:
please make it:
```
<property name="message" 
     value="mvn execution should be with '-e' option to produce error message in case of problem"/>
```

`\n` in message is not good. it makes output weird.

Reply:
done`